dart/welcome_email/README.md

```diff
@@ -0,0 +1,37 @@
+# 📧 Sending Welcome Emails using Mailgun's Email API
```

Comment:
Woah, really outdated readme. Does not include expected output and input data schema, building is totally, different and and deploy/event is appwrite related, so probably should not be in OR readme.